hack/test/e2e-run.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -e
+
+export DOCKER_ENGINE_GOARCH=${DOCKER_ENGINE_GOARCH:-amd64}
+
+echo "Ensure emptyfs image is loaded"
+bash /ensure-emptyfs.sh
+
+echo "Run integration/container tests"
+cd /tests/integration/container
+./test.main -test.v
```

Comment:
This script should probably make use of https://github.com/moby/moby/blob/master/hack/make/.integration-test-helpers#L25-L34 so that as we make changes to the moby/moby build this script wont break.

At the very least it should iterate over all the packages in that directory.

Reply:
I didn't use `.integration-test-helpers` because it depends on lots of other scripts. I will copy the functions used there though as they're cleaner than what I did.